internal/util/fileop.go

```diff
@@ -234,3 +242,63 @@ func resolvePath(path string, baseDir string, forceUnix bool) string {
 
 	return path
 }
+
+func handleGlobFileOperation(sourceGlob string, dest string, operation func(string, string) error) error {
+	sourcePaths, err := prepForGlobs(sourceGlob, dest)
+	if err != nil {
+		return fmt.Errorf("Failed to prep globs '%s' for an operation: '%w'.", sourceGlob, err)
```

Comment:
s/globs/glob path/
or
s/globs/glob pattern/

I would pick one and generally apply it throughout this PR.